run_mrm_services.py

```diff
@@ -0,0 +1,28 @@
+import sys
+from services.room_cancelation.auto_cancel_event import UpdateRecurringEvent
+from services.data_deletion.clean_deleted_data_from_db import DataDeletion
+
+services = {
+    "cleandb": DataDeletion().clean_deleted_data,
+    "autocancel_events": UpdateRecurringEvent().update_recurring_event_status
+}
+
+
+def run_mrm_services(arguments):
+    all_services = list(services.keys())
+    try:
+        if not arguments:
+            arguments = list(services.keys())
+        for argument in arguments:
+            services[argument]()
+    except KeyError as wrong_key:
+        print(
```

Comment:
Why do you prefer print statements in place of return statements?

Reply:
This is because it is a script that will be running on the terminal, hence the best way to inform the user of what is happening is to print it to the terminal.